src/content/docs/dropins-b2b/purchase-order/containers/approval-rule-details.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: ApprovalRuleDetails Container
+description: Learn about the ApprovalRuleDetails container in the Purchase Order drop-in.
+sidebar:
+  label: ApprovalRuleDetails
+---
+
+import { Aside } from '@astrojs/starlight/components';
+import TableWrapper from '@components/TableWrapper.astro';
+
+The ApprovalRuleDetails container displays and manages approval rule configuration including conditions, approvers, and rule metadata.
+
+<div style="background-color: var(--sl-color-blue-low); border-left: 4px solid var(--sl-color-blue); padding: 0.75rem 1rem; border-radius: 0.25rem; margin: 1rem 0;">
+<strong>Version: 1.0.0-beta2</strong>
+</div>
+
+## Configuration
+
+The `ApprovalRuleDetails` container provides the following configuration options:
+<TableWrapper nowrap={[0]}>
+
+| Parameter | Type | Req? | Description |
+|---|---|---|---|
+| `withHeader` | `boolean` | No | Whether to include the header |
+| `withWrapper` | `boolean` | No | Whether to include the wrapper |
+| `className` | `string` | No | Additional CSS classes to apply to the container |
```

Comment:
Global comment across all the containers. `className` is singular, but the descriptions says multiple classes can be specified. To me, the `string` type says one class is supported, but I don't know how arrays are handled.